src/nvim/syntax.c

```diff
@@ -7314,100 +7314,27 @@ static void highlight_attr_set_all(void)
  */
 int highlight_changed(void)
```

Comment:
The doc has some outdated parts now. 

Also `get_highlight_default()` is not used anywhere now.